manifests/0000_10_config-operator_01_proxy.crd.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: proxies.config.openshift.io
+spec:
+  group: config.openshift.io
+  scope: Cluster
+  versions:
+  - name: v1
+    served: true
+    storage: true
+  names:
+    kind: Proxy
+    listKind: ProxyList
+    plural: proxies
+    singular: proxy
+  validation:
```

Comment:
appears to be missing subresources, see https://github.com/openshift/cluster-config-operator/blob/master/manifests/0000_10_config-operator_01_image.crd.yaml#L17-L18 as an example.